app/tests/conftest.py

```diff
@@ -80,7 +80,7 @@ def db_client(monkeypatch_session) -> db.DBClient:
     """
 
     with db_testing.create_isolated_db(monkeypatch_session) as db_client:
-        models.metadata.create_all(bind=db_client.get_connection())
+        models.metadata.create_all(bind=db_client._engine)
```

Comment:
can we keep this to passing the connection so that we don't have to expose the the private property of the client? i'm reluctant to change the interface of DBClient to make the engine public just for the test suite – currently engine is private (hence the underscore prefix).

Reply:
What was there before doesn't work anymore, I'll see if there is another way to use the connection more directly.

Reply:
Looks like there is a way
```
        with db_client.get_connection() as conn, conn.begin():
            models.metadata.create_all(bind=conn)
```